mad-dog/src/helpers.js

```diff
@@ -131,7 +132,7 @@ async function _addUsers ({ userCount, executeAll, executeOne, existingUserIds,
         logger.info(`Existing users, userIds=${existingUserIds}`)
       } catch (e) {
         logger.error('GOT ERR CREATING USER')
-        logger.error(e.message)
+        console.error(e) // this prints out the stack trace
```

Comment:
can probably remove the logger.error above

Reply:
I left that logger.error in to print out the more mad-dog specific error. the console.error is for the stack trace. I think leaving it in will allow us to identify where in what flow in mad dog failed